new-dti-website/src/app/products/page.tsx

```diff
@@ -1,9 +1,146 @@
+import Image from 'next/image';
 import ImageCarousel from '../../../components/products/imageCarousel';
-import { products } from '../../../components/products/productIcons.json';
+import Connector from '../../../components/products/lines';
+import productIcons from '../../../components/products/productIcons.json';
+import productImages from '../../../components/products/productImages.json';
+import { Button } from '../../../components/ui/button';
+import { cn } from '../../../lib/utils';
 
 const Page = () => (
   <div>
-    <ImageCarousel items={products} />
+    <div className="flex lg:flex-row flex-col lg:max-w-[1186.76px] lg:mx-52 lg:my-24 lg:space-x-20 md:space-y-10 md:max-w-[698px] h-fit md:my-[130px] md:mx-[67px] max-w-[314px] mx-auto md:py-0 py-32">
```

Comment:
Talked to Vannessa, and the red blobs were intended on being placed exactly as they were in the design: https://cornelldti.slack.com/archives/C036VK3BRDE/p1727821358272699?thread_ts=1727820921.000559&cid=C036VK3BRDE